client/state/rewind/size/test/reducer.js

```diff
@@ -61,5 +65,21 @@ describe( 'rewind.size reducers', () => {
 				expect( sizeReducer( state, action ).retentionDays ).toEqual( expected );
 			}
 		);
+
+		it( 'should return the new retentionDays value if we set it using the action JETPACK_BACKUP_RETENTION_SET and state is undefined', () => {
+			const state = undefined;
+			const action = { type: JETPACK_BACKUP_RETENTION_SET, retentionDays: 30 };
+			const expected = 30;
+
+			expect( sizeReducer( state, action ).retentionDays ).toEqual( expected );
```

Comment:
Now that we're using `state` just once, we can just call it inline. Same for all other variables - `action` and `expected`. This makes the test quite short and straightforward, as you can see 😉

Reply:
Great, just made it shorter!